packages/client/src/create-deployment.ts

```diff
@@ -76,29 +72,6 @@ export default function buildCreateDeployment() {
 
     const { fileList } = await buildFileTree(path, clientOptions, debug);
 
-    let configPath: string | undefined;
-    if (!nowConfig) {
-      // If the user did not provide a config file, use the one in the root directory.
-      const relativePaths = fileList.map(f => relative(cwd, f));
-      const hasVercelConfig = relativePaths.includes('vercel.json');
-      const hasNowConfig = relativePaths.includes('now.json');
-
-      if (hasVercelConfig) {
-        if (hasNowConfig) {
-          throw new DeploymentError({
-            code: 'conflicting_config',
-            message:
```

Comment:
Why was this removed?

Reply:
`configPath` and `nowConfig` are never used throughout the function.

Reply:
so this whole block is useless; and thus the `nowConfig` function parameter is similarly useless.

Reply:
Strange because there is a test for this error message here

https://github.com/vercel/vercel/blob/b2bfae6e2e15237103607ab0f783617a0af1e45e/packages/cli/test/integration.js#L3210-L3229

Reply:
Yes because the if condition will still run when the config is omitted, but since that config is never used in this block I think the error message is pointless too. I will remove that test as well

Reply:
Upon review, this error is thrown from elsewhere in the code base. The test passes with no issue.